### DISCUSSION TITLE
How and where to set up environment variables ?

### DISCUSSION THREAD

**Discussion by Javier-Szyfer** [foundry-rs/foundry#1217] [Help] [answered]
Hi i'm new with foundry and been following the nft tutorial [here](https://book.getfoundry.sh/tutorials/solmate-nft.html) but it says to run commands directly on the CLI but this is confusing to me. How could i set up a RPC_URL and private key pairs for different networks.

Thanks

**Comment by onbjerg** [comment, marked as answer]
You would use a script for that, e.g. a bash script. More on environment variables [here](https://linuxhint.com/bash-environment-variables/)

**Reply by alymurtazamemon** [reply]
@onbjerg But this is not secure!

**Comment by PaulRBerg** [comment]
I recommend using [direnv](https://direnv.net/). Also, if are on macOS and you want to go the extra mile to set up your dot files once and for all, I recommend installing Mathias Bynens' [dotfiles](https://github.com/mathiasbynens/dotfiles).
